proposals/lip-0050.md

```diff
@@ -304,37 +400,53 @@ registerKeysParamsSchema = {
 ```python
 def verify(trs: Transaction) -> None:
 
-    validatorAddress = 20-byte address derived from trs.senderPublicKey
+    validatorAddress = SHA-256(trs.senderPublicKey)[:NUM_BYTES_ADDRESS]
 
-    if validator.getValidatorAccount(validatorAddress) does not exists :
```

Comment:
I realize that here, we might not even need this condition, just calling the API will do. If the validator address does not exist, the function will throw an error (and try to emit, but that will do nothing here).

Reply:
Ah, a very good point! Removed the check.